truss/tests/test_model_inference.py

```diff
@@ -974,3 +874,275 @@ def enable_gpu_fn(conf):
         # But make sure traces have parents at all.
         assert len(user_parents) > 3
         assert len(truss_parents) > 3
+
+
+# Returning Response Objects ###########################################################
```

Comment:
One thought -- do you think it makes sense to have an integration test for the stack trace we made in this PR?

Reply:
I was too lazy to lookup the traceback API and the initial implementation was totally bloated by chat GPT - I had this feeling that it could be simplified, now it's pretty straightforward.

But adding a test would be good too!